ups/pipe_base.table

```diff
@@ -1,8 +1,5 @@
-setupRequired(daf_butler)
-setupRequired(daf_persistence)
 setupRequired(log)
 setupRequired(pex_config)
 setupRequired(utils)
```

Comment:
If you are feeling particularly itchy about removing dependencies it may be possible to remove `lsst.utils` from the tests. The memory test case gives you nothing here and it would only be a problem if the tests use one of the utils helper methods (can't quickly check that).

Reply:
I have to admit I really don't have a clue about why essentially all the test_*.py files have a dummy `class MyMemoryTestCase(lsst.utils.tests.MemoryTestCase)` in them.  I'm sure there's something I'm missing there.